encord/orm/filter_preset.py

```diff
@@ -21,24 +21,28 @@ class FilterPreset(BaseDTO):
     last_updated_at: Optional[datetime] = Field(default=None, alias="lastUpdatedAt")
 
 
+class GetProjectFilterPresetParams(BaseDTO):
+    preset_uuids: Optional[List[uuid.UUID]] = Field(default=[], alias="uuids")
+    page_token: Optional[str] = Field(default=None, alias="pageToken")
+    page_size: Optional[int] = Field(default=None, alias="pageSize")
+
+
+class ProjectFilterPreset(BaseDTO):
+    preset_uuid: uuid.UUID = Field(alias="presetUuid")
+    name: str
+    created_at: Optional[datetime] = Field(default=None, alias="createdAt")
+    updated_at: Optional[datetime] = Field(default=None, alias="updatedAt")
+
+
 class FilterDefinition(BaseDTO):
     filters: List[Dict] = Field(default_factory=list)
 
 
 class FilterPresetDefinition(BaseDTO):
     local_filters: Dict[str, FilterDefinition] = Field(
-        default_factory=lambda: {str(uuid.UUID(int=0)): FilterDefinition()}, alias="local_filters"
+        default_factory=lambda: {str(uuid.UUID(int=0)): FilterDefinition()}, alias="localFilters"
```

Comment:
This is somewhat concerning that this changed. Same is shared between Active and Index and Index already used so this can't change.

Reply:
Checked that that change is compatible with master so we're chilling

Reply:
Actually we are camel-to-str on DTO anyway so this is un-necessary